server/src/modules/user/user.controller.ts

```diff
@@ -44,6 +44,7 @@ import {
 import { MapDto } from '@common/dto/map/map.dto';
 import { MapFavoriteDto } from '@common/dto/map/map-favorite.dto';
 import { MapLibraryService } from '@modules/maps/map-library.service';
+import { MapSummaryDto } from '@/common/dto/user/user-maps-summary.dto';
```

Comment:
Should be `@common` not `'@/common`. This works because `@/` is just `src/` but the idea with these named import paths is to avoid that hierarchy.